scripts/seed_data.py

```
@@ -0,0 +1,115 @@
+        return
+
+    try:
+        create_entities(data)
+        print("🎉 Data loaded successfully")
+    except (User.DoesNotExist, Classroom.DoesNotExist) as e:
+        print(f"❌ Entity not found: {str(e)}")
+    except KeyError as e:
+        print(f"❌ Missing required field: {str(e)}")
+    except ValueError as e:
+        print(f"❌ Invalid data format: {str(e)}")
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)